pkg/controllers/proxy/network_services_controller_test.go

```diff
@@ -187,12 +184,14 @@ var _ = Describe("NetworkServicesController", func() {
 			Expect(syncErr).To(Succeed())
 		})
 		It("Should have called cleanupMangleTableRule for ExternalIPs", func() {
+			fwmark1, _ := generateFwmark("1.1.1.1", "tcp", "8080")
+			fwmark2, _ := generateFwmark("2.2.2.2", "tcp", "8080")
```

Comment:
I think it would be better to catch grab the errors here and if there are any, then I would call `Fail` with the error